horovod/_keras/__init__.py

```diff
@@ -20,6 +20,7 @@
 import horovod.tensorflow as hvd
 import tensorflow as tf
 from horovod.common.gradient_aggregation import LocalGradientAggregationHelper
+from horovod.common.gradient_aggregation_v2 import LocalGradientAggregationHelperV2
 from horovod.common.tf_profile import TFProfileHelper
 
 
```

Comment:
I think we need to make this `_make_allreduce_grads_fn` look a bit more like `_DistributedOptimizer. _allreduce` and then delete `_DistributedOptimizer. _allreduce` since we aren't using it at the moment.